ee/clickhouse/queries/funnels/test/test_funnel.py

```diff
@@ -1012,21 +1034,21 @@ def test_funnel_step_breakdown_limit(self):
                     team=self.team,
                     event="sign up",
                     distinct_id=f"person_{num}_{i}",
-                    properties={"key": "val", "some_breakdown_val": f"{num}"},
+                    properties={"key": "val", "some_breakdown_val": num},
                     timestamp="2020-01-01T12:00:00Z",
                 )
                 _create_event(
                     team=self.team,
                     event="play movie",
                     distinct_id=f"person_{num}_{i}",
-                    properties={"key": "val", "some_breakdown_val": f"{num}"},
+                    properties={"key": "val", "some_breakdown_val": num},
                     timestamp="2020-01-01T13:00:00Z",
                 )
                 _create_event(
                     team=self.team,
                     event="buy",
                     distinct_id=f"person_{num}_{i}",
-                    properties={"key": "val", "some_breakdown_val": f"{num}"},
+                    properties={"key": "val", "some_breakdown_val": num},
```

Comment:
This enables testing for both, numeric and string values. Did a quick search on Metabase, and not everyone sends strings always